### ISSUE TITLE
OpenPaaS in layman terms?

### ISSUE THREAD

**Issue by amitguptagwl** [linagora/openpaas-esn#37]
Sorry to ask a very simple question. Can you please explain me what exactly OpenPassS does? I mean what problem do we solve?

What I understand, that may be wrong, is that OpenPass let you setup your own server hosting collaboration tools.

**Comment by abenchi** [comment]
Yes exactly. This is the power of opensource.

**Comment by amitguptagwl** [comment]
It's really cool thanks. Understanding what problem a system can solve in non-technical terms is really necessary for the people like me.

**Comment by nparquet** [comment]
You can get more non-technical information on what it does there : https://open-paas.org/

**Comment by mcarpenterjr** [comment]
Hey let's say we have email externally hosted, like on rackspace, can we integrate this per user into openPaas?

**Comment by ricardosaracino** [comment]
You can probably do it here 
![image](https://user-images.githubusercontent.com/759253/61163269-1ba37100-a4db-11e9-9444-1ac936178c19.png)
